docs/source/configuration.rst

```diff
@@ -59,16 +59,20 @@ You can specify configuration values in YAML files. For example:
 
 .. code-block:: yaml
 
-   array:
-      chunk-size: 128 MiB
-
-   distributed:
-      worker:
-         memory:
-            spill: 0.85  # default: 0.7
-            target: 0.75  # default: 0.6
-            terminate: 0.98  # default: 0.95
-
+  array:
+    chunk-size: 128 MiB
+
+  distributed:
+    worker:
+      memory:
+        spill: 0.85  # default: 0.7
+        target: 0.75  # default: 0.6
+        terminate: 0.98  # default: 0.95
+            
+    dashboard:
+      # Locate the dashboard if working on a server
+      link: /user/<user>/proxy/8787/status
```

Comment:
As you mention in the issue text, this isn't for just any server, but a JupyterHub server with `jupyter-server-proxy` (or `dask-labextension`) installed. See also https://github.com/dask/dask-labextension/issues/183 for some more requests for better documentation of this.

So, for a Jupyter server *not* using JupyterHub, this would just be `/proxy/8787/status`